src/commands/cmd_string.cc

```diff
@@ -86,6 +86,8 @@ class CommandGetEx : public Commander {
       uint32_t max_btos_size = static_cast<uint32_t>(config->max_bitmap_to_string_mb) * MiB;
       redis::Bitmap bitmap_db(svr->storage, conn->GetNamespace());
       s = bitmap_db.GetString(args_[1], max_btos_size, &value);
```

Comment:
Seems that L79 might find that `GetEx` is unexists, and then return. However when key is an expired Bitmap or having other error, we might not `Expire` it.

```
s = bitmap_db.GetString(args_[1], max_btos_size, &value);
if (s.ok()) {
  // ...
}
```

Better change to this?